mteb/models/gritlm_models.py

```diff
@@ -13,12 +13,7 @@
 
 GRIT_LM_TRAINING_DATA = {
     **E5_TRAINING_DATA,  # source https://arxiv.org/pdf/2402.09906
-    # also uses medi2 which contains fever and hotpotqa:
-    "FEVER": ["train"],
```

Comment:
doesn't it appear in Table 29?

Reply:
Sorry fixed it - There was a bug with E5_TRAINING_DATA being used for e5-mistral thus I reused that one here but should have been E5_MISTRAL_..